USAGE_DATA.md

```diff
@@ -13,6 +13,7 @@ vscode-quarkus has opt-in telemetry collection, provided by [vscode-commons](htt
    The following commands emit these telemetry events:
     * "Quarkus: Add extensions to current project"
     * "Quarkus: Debug current Quarkus project"
+    * "Quarkus: Deploy current Quarkus Project to OpenShift"
```

Comment:
I'd probably update this to reflect the changes made in https://github.com/redhat-developer/vscode-quarkus/pull/374 . Here and in the README.